spec/lib/tasks/invalid_activities_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+require "csv"
+Rails.application.load_tasks
+
+describe "invalid_activities.rake" do
+  it "creates a csv file on tmp folder with a list of invalid activities in RODA" do
+    activities = create_list(:project_activity, 5)
+    activities.first.update_columns(gdi: nil, collaboration_type: nil)
```

Comment:
I'm interested to learn what led to you choosing `update_columns` here, I've never seen it?!

Reply:
I learnt about `update_columns` recently too (thanks to Leeky). Here I try to modify two activities to make them invalid by changing the values of some mandatory fields to `nil`. I tested on Rails console first, and the usual methods for updating these fields were failing when I tried to save the activity (the validation rules made these changes fail and the DB got rollback). `update_columns` will allow me to change the values of those columns because you do it in the database directly, bypassing validations and callbacks.

https://apidock.com/rails/ActiveRecord/Persistence/update_columns